src/ScriptEditor/NetscriptDefinitions.d.ts

```diff
@@ -3222,7 +3222,12 @@ export interface CodingContract {
    * @param opts - Optional parameters for configuring function behavior.
    * @returns True if the solution was correct, false otherwise. If the returnReward option is configured, then the function will instead return a string. If the contract is successfully solved, the string will contain a description of the contract’s reward. Otherwise, it will be an empty string.
    */
-  attempt(answer: string[] | number, filename: string, host?: string, opts?: CodingAttemptOptions): boolean | string;
+  attempt(
```

Comment:
Previous documentation was just wrong.
`string` is always an accepted type of input. Other types that would be expected are `number`, `number[]`, `number[][]` `string[]` but that would look ridiculous so any[] covers all the array types.